index.js

```diff
@@ -236,12 +236,22 @@ export default class OneSignal {
     static sendTag(key, value) {
         if (!checkIfInitialized()) return;
 
+        if (typeof value !== "string") {
+            value = JSON.stringify(value);
```

Comment:
I am not sure how the bridge treats `object` today. If we stringify everything here then it could result in some large objects sending a huge object KiBs in size. Let's limit it to just `boolean` of the bug we are fixing.